core/src/server/instance-manager.ts

```diff
@@ -160,16 +161,17 @@ export class GardenInstanceManager {
         log.debug(`Instance key: ${key}`)
 
         garden = await Garden.factory(params.projectRoot, {
-          // The sessionId should be the same as the surrounding process.
-          // For each command run, this will be set as the parentSessionId,
-          // and the command-specific Garden (cloned in `Command.run()`) gets its own sessionId.
-          sessionId: this.sessionId,
           monitors: this.monitors,
           plugins: this.plugins,
           ...this.defaultOpts,
           ...garden?.opts,
           ...opts,
           ...params,
+          // The sessionId should be the same as the surrounding process.
+          // For each command run, this will be set as the parentSessionId,
+          // and the command-specific Garden (cloned in `Command.run()`) gets its own sessionId.
+          sessionUlid: this.sessionUlid,
+          parentSessionUlid: null,
```

Comment:
I wonder if this should be 
```suggestion
          parentSessionUlid: this.parentSessionUlid,
```
Probably a theoretical question..

Reply:
I'll take a look

Reply:
There is no field `parentSessionUlid` defined in `GardenInstanceManager`, so the `null` looks ok here.